Walkthroughs/ImageWhiteout/README.md

```diff
@@ -0,0 +1,215 @@
+longer see them if you delete them in a child image).
+
+### Union Filesystems
+Simply put, a union filesystem creates a cohesive view of a filesystem that is physically composed of several different
+groups of files (layers, branches, directories, etc). The driver is responsible for presenting the filesystem to the
+system and handles all of the mechanics of presenting the merged view.
+
+The mostly commonly used filesystem in Docker currently is the 
+[OverlayFS](https://github.com/torvalds/linux/tree/master/fs/overlayfs), which is what we will be using for this 
+example.
+
+## The Problem; Deleted Files...are not Deleted
+Remember the good old days of reaching for that little bottle of whiteout to cover up a mistake on paper? 
+Docker's OverlayFS operates on a similar principle, but in the digital realm. Just like how whiteout conceals the error
+beneath a white layer, allowing for the possibility to reveal it again by scraping it off, OverlayFS works by layering 
+directories - a lower one that is read-only and an upper one that is read-write. When Docker pulls an image, it fetches 
+a series of read-only layers (the original text, if you will) and stacks them neatly one on top of the other. Now, when 
+you run a container and make changes (or mistakes), these alterations occur in a top, writeable layer, essentially 
+using digital "whiteout" to mask the elements in the layers below without altering them. This means, much like peeling 
+back dried whiteout, you can still access the original details in the lower layers if needed. 
+
+Although this is efficient from a technical and organizational standpoint, it does have the side effect allowing access 
+to a file you don't really want people to have access to. Worse, many people do not fully understand how the OverlayFS 
+works so they are blissfully unaware of their potential exposure.
+
+### How Tough is it to Get the File Back?
+Not really tough at all; remember that a docker image is built from a number of layers. So it's just a matter of 
+finding the layer where the file you want to recover was created and pulling it from there.
+
+### Example?
+Sure, let's go through a quick example using a small image with a handful of layers.
+
+#### Dockerfile
+This is a simple dockerfile that pulls the latest Ubuntu image, sets the work location to `/root`, does an update, 
+and copies over a secret password. Then it deletes the password.
+
+```dockerfile
+FROM ubuntu
+WORKDIR /root
+RUN apt update && apt install -y nodejs
+COPY MySecretPassword .
+RUN rm -f MySecretPassword
+```
+
+```shell
+$ cat MySecretPassword
+Password
+```
+
+#### Build The Image
+Build the image.
+
+```shell
+$ docker buildx build --no-cache -t password-test .
+
+[+] Building 10.0s (10/10) FINISHED                                                                                                  docker:desktop-linux
```

Comment:
Is there any important info that we are highlighting from the build output? If not, may be we can remove it.

Reply:
I'm of two minds on that - on the one hand it takes up space, but on the other hand if someone is struggling it may help to see what the output looks like. 

My ideal solution would be to have disclosure widgets so we could collapse them down as a default.